src/validation.cpp

```diff
@@ -518,18 +516,16 @@ class MemPoolAccept
         // Private ctor to avoid exposing details to clients and allowing the possibility of
         // mixing up the order of the arguments. Use static functions above instead.
         ATMPArgs(const CChainParams& chainparams,
-                 int64_t accept_time,
-                 bool bypass_limits,
+                 const int64_t accept_time,
+                 const std::optional<MemPoolBypass>& mempool_bypass,
```

Comment:
Probably better to just have a constexpr noop `MemPoolBypass`

Reply:
Could you elaborate more ? I don´t see how `constexpr` fits in this case.